src/Microsoft.Extensions.Http/Logging/EventIds.cs

```diff
@@ -23,5 +23,10 @@ internal static class EventIds
         public static readonly EventId ClientHandlerRequestHeader = new EventId(102, "RequestHeader");
         public static readonly EventId ClientHandlerResponseHeader = new EventId(103, "ResponseHeader");
 
+        // Logging for the DefaultHttpClientFactory
+        public static readonly EventId FactoryCleanupCycleStart = new EventId(100, "CleanupCycleStart");
+        public static readonly EventId FactoryCleanupCycleEnd = new EventId(101, "CleanupCycleEnd");
+        public static readonly EventId FactoryCleanupItemFailed = new EventId(102, "CleanupItemFailed");
+        public static readonly EventId FactoryHandlerExpired = new EventId(103, "HandlerExpired");
```

Comment:
Hrm, didn't notice this fully in previous reviews. Why is this not part of the `Log` nested class? Aren't we just substituting one large comment-delimited class for another? Also, putting them in `Log` means the names can use `nameof`.

Reply:
yeah I think you're right about this. I'm not going to use nameof though. I want these to be hardcoded constants because we're not allowed to change them.

Reply:
> I want these to be hardcoded constants because we're not allowed to change them.

Good point.